osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/ws/DefaultWebServiceTemplateFactory.java

```diff
@@ -87,6 +91,7 @@ public WebServiceTemplate getTemplate(final String organisationIdentification, f
         private int maxConnectionsTotal = 20;
         private int connectionTimeout = 120000;
         private CircuitBreaker circuitBreaker;
+        private WebServiceTemplateHostnameVerificationStrategy webServiceTemplateHostnameVerificationStrategy = WebServiceTemplateHostnameVerificationStrategy.ALLOW_ALL_HOSTNAMES;
```

Comment:
Make sure that components not yet implementing the `BROWSER_COMPATIBLE_HOSTNAMES` hostname verification are constructed with `ALLOW_ALL_HOSTNAMES`.